docusaurus/docs/reference/tunnelers/linux/index.md

```diff
@@ -92,27 +98,17 @@ sudo apt install ziti-edge-tunnel
 
 2. Place a copy of the enrollment token JWT file in the `/opt/openziti/etc/identities` directory.
```

Comment:
I feel like this should be a "see below for adding identities" where "identities" will be explained.